diff.go

```diff
@@ -2,10 +2,16 @@ package reviewdog
 
 import (
 	"context"
+	"fmt"
 	"os/exec"
 	"sync"
 )
 
+var Y = fmt.Sprintf("test change %s")
+
+func unused() {
```

Comment:
**[unused]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
func unused is unused (U1000)